.vuepress/config.js

```diff
@@ -134,6 +134,7 @@ module.exports = {
     ['vuepress-plugin-container', { type: 'note' }],
     [require('./config-path-checker')],
     [require('./custom-markdown-rules')],
+    [require('./code-samples/samplesPlugin')],
```

Comment:
a plugin with the word plugin in its name is probably unnecessary.

Reply:
I have changed it to index.js :)